docs/ctk-configurations.md

```diff
@@ -0,0 +1,33 @@
+## compliance toolkit Configurations
+| Configuration   |      Description      |  Default Value |
+|-----------------|-------------------|---------------|
+| `mosip.toolkit.sbi.ports`| Default range of SBI (Secure Biometric Interface) ports scanned to connect to the devices. | 4501,4502,4503,4504,4505,4506,4507,4508,4509,4510 |
+| `mosip.toolkit.sbi.timeout`|  This specifies the default timeout value for all SBI testcases which do not include 'timeout' in 'otherAttributes'. | 10000 |
+| `mosip.toolkit.sbi.keyrotation.iterations`| This specifies the number of times the device key has to be rotated for execution of SBI testcases : SBI1022, SBI1023, SBI1024, SBI1025, SBI1060, SBI1061 | 2 |
+| `mosip.toolkit.sbi.timestamp-interval`| Specifies the time interval, in minutes, in which SBI device should send back the response, verified through testing with the following SBI test cases : SBI1083, SBI1084, SBI1085, SBI1086, SBI1087, SBI1088, SBI1089, SBI1090 | 3 |
+| `mosip.toolkit.languages.rtl`| Specifies the languages that follow a right-to-left (RTL) script direction. | ara |
+| `mosip.toolkit.rcapture.encryption.enabled`| Enables encryption of 'bioValue' field of the SBI rcapture response before saving data into the database for RCapture SBI testcases. | true |
+| `mosip.toolkit.sdk.finger.qualitycheck.threshold.value`| Specifies the minimum quality acceptance threshold value, used for quality checking when processing finger biometric data using SDK. | 60 |
+| `mosip.toolkit.sdk.face.qualitycheck.threshold.value`| Specifies the minimum quality acceptance threshold value, used for quality checking when processing face biometric data using SDK. | 30 |
```

Comment:
Specifies the minimum quality acceptance threshold value' Used for quality checking when processing face biometric data using SDK.